cli/tests/integration/run_tests.rs

```diff
@@ -2197,9 +2197,9 @@ mod permissions {
     let args = "run --quiet 061_permissions_request.ts";
     use util::PtyData::*;
     util::test_pty2(args, vec![
-      Output("⚠️  ️Deno requests read access to \"foo\". Allow? [y/n (y = yes allow, n = no deny)] "),
+      Output("⚠️  ️Deno requests read access to \"foo\". Run again with --allow-read to bypass this prompt.\r\n   Allow? [y/n (y = yes allow, n = no deny)] "),
```

Comment:
I don't understand why some of these have a trailing space and others don't considering a trailing space was added to the output. I guess there's some trimming on the output being done in the comparison.